magento/magento2-page-builder
# Alt text for Images, Banners and Slides for better accessibility and SEO

### Description (*)
* SEO Fieldset in Page Builder **Image / Banner / Slide**
![seo-fieldset-in-page-builder-form](https://user-images.githubusercontent.com/52840038/136785061-d11bd765-a124-4bf1-899a-76995d41b636.png)
*  Banner SEO in frontend
![banner-seo-proof](https://user-images.githubusercontent.com/52840038/136785751-fcd45a69-7e4d-4824-9451-1b4355c45d30.jpg)

* Slider SEO in Frontend
![slide-seo-proof](https://user-images.githubusercontent.com/52840038/136785801-d620bd02-ef2c-4185-87cc-0951636e808f.jpg)

* Image is already having SEO functionality


### Story

* [746](https://jira.corp.magento.com/browse/746) Alt text for Images


### Fixed Issues (if relevant)

1. magento/magento2-page-builder#746: Alt text for Images

### Manual testing scenarios (*)
* By going inside `Admin Panel -> CONTENT -> Pages -> Edit Home page` open Content section and click on **Edit with Page Builder** button.
 * Add New Banner/Add New Slider **->** Hover on the Banner Area / Edit Slide **->** Click On ⚙️[Edit] Here admin can find **Search Engin Optimization** Fieldset.
 * Inside **Search Engin Optimization** Fieldset **Alternative Text** and **Title Attribute** fields will be there.
 * After filling required fields, One can check by going to the `Storefront -> Home page`  and inspect **Banner / Slide / Image** and check that Alternative Text and Title Attribute will be there which makes PageBuilder content accessible and SEO optimized 

### Questions or comments


### Checklist
